code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -91,7 +91,7 @@
 	update_icon()
 
 /obj/item/gun/ballistic/automatic/wt550
-	name = "security auto rifle"
+	name = "security PDW"
```

Comment:
why

Reply:
the description says PDW, and it is a PDW.

Reply:
but auto rifle sounds much cooler